<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_openFile">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Otw√≥rz plik</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_chooseCSV">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Wybierz plik .csv</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>20</width>
     <height>591</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_table">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Tabela</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_table">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>261</width>
     <height>521</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_roundTeam">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>80</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_RoundNumber">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>80</y>
     <width>48</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_roundResult">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>120</y>
     <width>331</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_goalsScoredConceded">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>200</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_goalsScoredConceded">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>230</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
